frontends/concrete-python/concrete/fhe/mlir/utils.py

```diff
@@ -195,3 +200,33 @@ def construct_deduplicated_tables(
     return tuple(
         (hashable_array.array, indices) for hashable_array, indices in tables_to_cell_idx.items()
     )
+
+
+# replace missing tensor.FromElementsOp.__init__
+class _FromElementsOp(tensor.FromElementsOp):
+    def __init__(self, result: MlirType, *elements: MlirOperation):
+        if not isinstance(result, MlirType):  # pragma: no cover
+            msg = (
+                "_FromElementsOp(result, ...), result, actual: "
+                f"{type(result)}, expected: {MlirType}"
+            )
+            raise TypeError(msg)
```

Comment:
Let's make this an assertion, which is for things users should never see, so it's appropriate

Reply:
ok